build/RELEASING.md

```diff
@@ -130,12 +146,12 @@ Only for Maintainers.
 14. Packages would be available in nuget.org in few minutes. Validate that the
     package is uploaded.
 
-15. Delete the API key generated in Step 11.
+15. Delete the API key generated in Step 12.
 
 16. Update the OpenTelemetry.io document
     [here](https://github.com/open-telemetry/opentelemetry.io/tree/main/content/en/docs/net)
     by sending a Pull Request.
 
-17. If a new stable version of the core packages were released, update
-    `OTelLatestStableVer` in Directory.Packages.props to the just released
-    stable version.
+17. If a new stable version of the core packages were released, open a PR to
+    update the `OTelLatestStableVer` property in Directory.Packages.props to the
+    just released stable version.
```

Comment:
Q: Do you want to upgrade on the contrib side should be part of the release process?
I would at least add creating an issue to track in the contrib repository.

Reply:
Added a step for creating an issue on the contrib side. Good idea thanks!